web-wallet/src/lib/components/__tests__/OperationResult.spec.js

```diff
@@ -1,117 +1,127 @@
-import {
-	afterEach,
-	describe,
-	expect,
-	it,
-	vi
-} from "vitest";
+import { afterEach, describe, expect, it, vi } from "vitest";
 import { cleanup, render } from "@testing-library/svelte";
 
 import { OperationResult } from "..";
 
 vi.useFakeTimers();
 
 describe("OperationResult", () => {
-	const delay = 1000;
+  const delay = 1000;
 
-	/** @type {(delay: number) => Promise<any>} */
-	const rejectAfter = ms => new Promise((resolve, reject) => {
-		setTimeout(() => reject(new Error("some error")), ms);
-	});
+  /** @type {(delay: number) => Promise<any>} */
+  const rejectAfter = (ms) =>
```

Comment:
Not related to the PR, but still...

There's a mismatch between the function param and the doc comment above (`delay` vs `ms`).

Reply:
> Not related to the PR, but still...
> 
> There's a mismatch between the function param and the doc comment above (`delay` vs `ms`).

Doesn't the type checker usually show an error for this?

Reply:
Opened issue https://github.com/dusk-network/rusk/issues/1471.